docker/prognostic_run/Dockerfile

```diff
@@ -79,4 +79,13 @@ FROM bld AS prognostic-run
 RUN make -C docs html
 
 ARG COMMIT_SHA_ARG
-ENV COMMIT_SHA=$COMMIT_SHA_ARG
\ No newline at end of file
+ENV COMMIT_SHA=$COMMIT_SHA_ARG
+
+
+###############################################################################
+#  additionally, with radiation port available
+###############################################################################
+FROM prognostic-run AS prognostic-run-radiation
```

Comment:
What is the downside of including it in the normal image? It could be confusing to have too many prognostic_run images and related makefile targets.

Reply:
Not much, since there aren't really any requirements differences. I'll make them one and the same.